src/v2/api/controllers/AuthController.ts

```diff
@@ -160,13 +176,30 @@ export class AuthController {
     return this.userService.getUser(req.user.id);
   }
 
+  @ApiOkResponse()
+  @ApiBadRequestResponse({
```

Comment:
There are 2 InvalidBodyExceptions in ForgotPasswordDTO, describe them

Reply:
Done